feelpp/feel/feelmodels/modelproperties.cpp

```diff
@@ -89,6 +89,7 @@ ModelProperties::ModelProperties( std::string const& filename, std::string const
         if ( Environment::isMasterRank() )
             std::cout << "Missing ShortName entry in model properties - set it to the Name entry : " << M_name << "\n";
     }
+    M_unit = M_p.get( "Unit", "m" );
```

Comment:
@romainhild It's not really usable, we need to have a Units section with subentry as length, mass, temperature, time, ... 
```
Units:
{
"length":mm",
"time":"ms", 
...
}
```
There is also the choice of the reference system : SI, CGS, ...

Reply:
I agree with you but couldn't we keep it for now, and open an issue to improve it later ?
I already use it in one of my code.